moment-range/index.d.ts

```diff
@@ -7,40 +7,50 @@ import * as moment from 'moment';
 
 export = moment;
 
+type interval = 'years' | 'quarters' | 'months' | 'weeks' | 'days' | 'hours' | 'minutes' | 'seconds' | 'milliseconds';
+
 declare module 'moment' {
      interface Range {
         start: Moment;
         end: Moment;
 
-        contains (other: Date, exclusive?: boolean): boolean;
-        contains (other: Moment, exclusive?: boolean): boolean;
+        adjacent (other: Range): boolean;
```

Comment:
can you not change the order of declarations please. it makes it hard to understand what was the change.